packages/flutter/lib/src/widgets/editable.dart

```diff
@@ -212,14 +274,17 @@ class RawEditableTextState extends State<RawEditableLine> {
     else if (!config.focused && _cursorTimer != null)
       _stopCursorTimer();
 
-    return new _EditableLineWidget(
-      value: config.value,
-      style: config.style,
-      cursorColor: config.cursorColor,
-      showCursor: _showCursor,
-      hideText: config.hideText,
-      onContentSizeChanged: config.onContentSizeChanged,
-      scrollOffset: config.scrollOffset
+    return new SizeObserver(
```

Comment:
I just moved it from one place to another.  It's the same problem that all the scrollables have about wanting to know how big their outsides are for the scrolling physics.